introspection-engine/connectors/sql-introspection-connector/src/misc_helpers.rs

```diff
@@ -83,22 +83,22 @@ fn common_prisma_m_to_n_relation_conditions(table: &Table) -> bool {
 //calculators
 
 pub fn calculate_many_to_many_field(
-    foreign_key: &ForeignKey,
+    opposite_foreign_key: &ForeignKey,
     relation_name: String,
     is_self_relation: bool,
 ) -> RelationField {
     let relation_info = RelationInfo {
         name: relation_name,
         fields: vec![],
-        to: foreign_key.referenced_table.clone(),
-        to_fields: foreign_key.referenced_columns.clone(),
+        to: opposite_foreign_key.referenced_table.clone(),
+        to_fields: opposite_foreign_key.referenced_columns.clone(),
```

Comment:
Is this correct? I thought we were supposed to have the prisma-level names here, not the database names (in case the field names are remapped).

Reply:
Yes that should be prisma level names indeed.

Reply:
But it might not be a problem after all. We construct the datamodel only to be able to render it to a string in the end. During rendering we do not render the `to_fields` at all. So you would never be able to observe that this is incorrect.